python/pyspark/sql/types.py

```diff
@@ -1300,70 +1300,80 @@ def _verify_type(obj, dataType, nullable=True):
         if nullable:
             return
         else:
-            raise ValueError("This field is not nullable, but got None")
+            raise ValueError("%s: This field is not nullable, but got None" % name)
```

Comment:
Probably, I missed something. However, is there any test case that actually checks this message change?